lib/tool_belt/render.rb

```diff
@@ -0,0 +1,10 @@
+require 'erb'
+require 'ostruct'
+
+def render(template, vars)
+  ERB.new(template).result(OpenStruct.new(vars).instance_eval { binding })
+end
+
+def render_file(filename, context)
+  render(File.read(filename), context)
+end
```

Comment:
I'm not a huge fan of unscoped functions. That feels like dirty Ruby compared to an class or module to handle this.

Reply:
There may be value in these in the future inside a Template class or module for other actions.